crates/webidl/src/first_pass.rs

```diff
@@ -115,6 +116,22 @@ pub(crate) struct Arg<'src> {
     pub(crate) variadic: bool,
 }
 
+#[derive(Clone, Debug)]
+pub(crate) enum Iterable<'src> {
+    MapLike {
+        key: weedle::types::Type<'src>,
+        value: weedle::types::Type<'src>,
+    },
+    ArrayLike(weedle::types::Type<'src>),
```

Comment:
I think the types here can probably be removed?

Reply:
Yeah I left them in so that in future they could be used to generate rust types, but yep they could be removed now. Shall I remove them?